homework-18/main.js

```diff
@@ -0,0 +1,119 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const form = document.querySelector('form');
+    const search = document.querySelector('.header-search');
+    const moviesEl = document.querySelector('.movies');
+    const paginationEl = document.querySelector('.pagination');
+    const API_KEY = 'f67b2711';
+    let currentPage = 1;
+    let rows = 10;
+    form.addEventListener('submit', (e) => {
+        e.preventDefault();
+        getPage();
+
+    })
+
+    async function getPage(page) {
+        const resp = await axios.get(`http://www.omdbapi.com/?apikey=${API_KEY}&s=${search.value}&page=${page}`);
+        const respData = resp.data;
+        showMovies(respData);
+        displayPagination(respData.totalResults, rows);
+    }
+
+    function showMovies(data) {
+        moviesEl.innerHTML = '';
+        paginationEl.innerHTML = '';
+        if (!data.Search) {
+            moviesEl.innerHTML = "Sorry we cannot find movie";
+        }
+
+        data.Search.forEach(movie => {
+            const movieEl = document.createElement("div");
+            movieEl.classList.add("movie");
+            movieEl.innerHTML = `
+        <div class="movie">
+        <div class="movie-cover-inner">
+            <img src="${movie.Poster}"
+                class="movie-cover" alt="${movie.Title}" />
+            <div class="movie-cover-darkened">
+            </div>
+        </div>
+        <div class="movie-info">
+            <div class="movie-title">${movie.Title}</div>
+            <div class="movie-year">${movie.Year}</div>
+        </div>
+    </div>`;
+            movieEl.addEventListener('click', () => openModal(movie.imdbID));
+            moviesEl.appendChild(movieEl);
+
+        });
+
+    }
+    const modalEl = document.querySelector(".modal");
+    async function openModal(id) {
+        const resp = await axios.get('http://www.omdbapi.com/?apikey=f67b2711&i=' + id);
+        const respData = resp.data;
+        modalEl.classList.add('modal-show');
+        modalEl.innerHTML = `
+<div class="modal-card">
+    <img class="modal-movie-backdrop" src="${respData.Poster}" alt="">
```

Comment:
There was an issue with the poster in the [previous homework](https://github.com/mariianazarova/Hillel.JS.Pro/pull/15/files#r1036944305), it should have a fallback image, but it  isn't there yet